sphinx_scality/static/css/current.css_t

```diff
@@ -297,4 +310,109 @@ input[type=text], select {
 .docnames {
     font-size: 10px;
     font-color: #a9a9a9;
-}
\ No newline at end of file
+}
+
+/* TITLES */
+
+div.body h1 { margin-top: 0; font-size: 200%; }
+div.body h2 { font-size: 160%; }
+div.body h3 { font-size: 140%; }
+div.body h4 { font-size: 120%; }
+div.body h5 { font-size: 110%; }
+div.body h6 { font-size: 100%; }
+
+/* Admonition boxes */
+
+div.important {
+    background-color: #ffffe0;
+    border: 1px solid #ffdc50;
+}
+
+div.tip {
+    background-color: #eeffee;
+    border: 1px solid #93d893;
+}
+
+div.note {
+    background-color: #eeeeff;
+    border: 1px solid #aeaee2;
+}
+
+div.warning {
+    background-color: #ff9b3b;
+    border: 1px solid #e55529;
+}
+
+div.admonition.warning a:visited {
+    color: #3F5952;
+    text-decoration: none;
+}
+
+/* CODE */
+
+pre
+  {   font-size: 90%;
+      font-family: Courier, "Courier New", monospace ; 
+/*      margin-left: 18pt;*/
+      margin-top: 0pt;
+      color: '#000000';
+      background-color: '#F2F2F2';
+      border-bottom: none;      
+      border-top: none;
+      border-left: none;
+      border-right: none;
+  }
+
+/* TABLES */
+
+table.docutils th {
+  background-color: #eaeaea;
+}
+
+table caption {
+    font-weight: bold;
+    text-align: left;
+    margin-top: 20px;
+}
+
+table.docutils td, table.docutils th {
+    border-top: 1px solid #aaa;
+}
+
+/* IMAGES */
+
+img
+  {
+    align: center;
+    padding-left: 10px;
+    padding-right: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+  }
+
+
+img.FiftyPercent
```

Comment:
What are those class names? Did we introduce them? Because they are quite non-standard.